GeometryService/src/STMMaker.cc

```diff
@@ -376,15 +461,22 @@ namespace mu2e {
     _SSCollimatorHole1xOffset     = _config.getDouble("stm.SScollimator.hole1.xoffset");
     _SSCollimatorHole1RadiusUpStr = _config.getDouble("stm.SScollimator.hole1.radiusUpStr");
     _SSCollimatorHole1RadiusDnStr = _config.getDouble("stm.SScollimator.hole1.radiusDnStr");
-    _SSCollimatorHole1LinerBuild     = _config.getBool(  "stm.SScollimator.hole1.liner.build");
-    _SSCollimatorHole1LinerThickness = _config.getDouble("stm.SScollimator.hole1.liner.thickness");
+		_SSCollimatorHole1Liner1Build     = _config.getBool(  "stm.SScollimator.hole1.liner1.build");//wyq
+		_SSCollimatorHole1Liner1Thickness = _config.getDouble("stm.SScollimator.hole1.liner1.thickness");//wyq
+		_SSCollimatorHole1Liner2Build     = _config.getBool(  "stm.SScollimator.hole1.liner2.build");//wyq
+		_SSCollimatorHole1Liner2Thickness = _config.getDouble("stm.SScollimator.hole1.liner2.thickness");//wyq
     _SSCollimatorHole2Build       = _config.getBool(  "stm.SScollimator.hole2.build");
     _SSCollimatorHole2xOffset     = _config.getDouble("stm.SScollimator.hole2.xoffset");
     _SSCollimatorHole2RadiusUpStr = _config.getDouble("stm.SScollimator.hole2.radiusUpStr");
     _SSCollimatorHole2RadiusDnStr = _config.getDouble("stm.SScollimator.hole2.radiusDnStr");    
-    _SSCollimatorHole2LinerBuild     = _config.getBool(  "stm.SScollimator.hole2.liner.build");
-    _SSCollimatorHole2LinerThickness = _config.getDouble("stm.SScollimator.hole2.liner.thickness");
-    _SSCollimatorHoleLinerMaterial = _config.getString("stm.SScollimator.hole.liner.material");       
+		_SSCollimatorHole2Liner1Build     = _config.getBool(  "stm.SScollimator.hole2.liner1.build");//wyq
+		_SSCollimatorHole2Liner1Thickness = _config.getDouble("stm.SScollimator.hole2.liner1.thickness");//wyq
+		_SSCollimatorHole2Liner2Build     = _config.getBool(  "stm.SScollimator.hole2.liner2.build");//wyq
+		_SSCollimatorHole2Liner2Thickness = _config.getDouble("stm.SScollimator.hole2.liner2.thickness");//wyq
+		_SSCollimatorHole1Liner1Material = _config.getString("stm.SScollimator.hole1.liner1.material");       //wyq
+		_SSCollimatorHole1Liner2Material = _config.getString("stm.SScollimator.hole1.liner2.material");       //wyq
+		_SSCollimatorHole2Liner1Material = _config.getString("stm.SScollimator.hole2.liner1.material");       //wyq
+		_SSCollimatorHole2Liner2Material = _config.getString("stm.SScollimator.hole2.liner2.material");       //wyq
```

Comment:
Should this default to run in the original manner (no 2 layers) if liner1 and liner2 aren't specified?